include/Screens/ControlsScreen.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef CONTROLSSCREEN_HPP
+#define CONTROLSSCREEN_HPP
+
+#include "Global.hpp"
+#include "Screens/Screen.hpp"
+#include <sstream>
+#include <fstream>
+#include <string>
+#include <cmath>
+
+class ControlsScreen : public Screen {
+public:
+	ControlsScreen();
+	void render(sf::RenderWindow *window) override;
+	void interpretInput(sf::Event event) override;
+
+
+    //loads current key bindings
+    bool loadKeys();
+
+	bool saveKeys();
```

Comment:
this doesn't look aligned on github :'( dw about changing it but is why I prefer spaces > tabs